packages/nodejs/src/config.ts

```diff
@@ -48,6 +43,34 @@ export class Configuration {
     return true
   }
 
+  /**
+   * Explicit default configuration values
+   *
+   * @private
+   */
+  private _defaultValues(): { [key: string]: any } {
+    return {
+      caFilePath: path.join(__dirname, "../cert/cacert.pem"),
+      debug: false,
+      dnsServers: [],
+      enableHostMetrics: true,
+      enableMinutelyProbes: true,
+      enableStatsd: false,
+      endpoint: "https://push.appsignal.com",
+      environment: process.env.NODE_ENV || "development",
+      filesWorldAccessible: true,
+      filterDataKeys: [],
+      filterParameters: [],
+      filterSessionData: [],
+      ignoreActions: [],
+      ignoreErrors: [],
+      ignoreNamespaces: [],
+      log: "file",
+      logPath: "/tmp",
+      transactionDebugMode: false
```

Comment:
I don't think this is everything. Some options I see missing (but I may be missing some):

- send_params: true
- skip_session_data: false
- request_headers: `[]` (we supply a list of headers in the other integrations.

It could also be there are not completely implemented in this integration? I can find some references to them in the `configmap.ts` file, but I can't find them being used anywhere. If they're not properly implemented, let's create issues for them to fix that and we can merge this PR as is.

Reply:
None of them are present in the docs. I'll open issues for them cause as you say, they're probably not properly implemented.